docs/wiki/The CI environment - Token replacement.md

```diff
@@ -67,20 +67,19 @@ Let us say you'd want to use this token inside a Key Vault parameter file, to de
 
 Once the Key Vault is deployed, you'll notice that the Key Vault name in Azure will be `tokenValue-keyVault`
 
-The token prefix `'<<'` and suffix `'>>'` in the above example are also configurable in the [Settings.json](https://github.com/Azure/ResourceModules/blob/main/settings.json) file. They are however the default used in the CARML main repository.
+The token prefix `'<<'` and suffix `'>>'` in the above example are also configurable in the [Settings.json](https://github.com/Azure/ResourceModules/blob/main/settings.json) file.
 
-> **Note**: The solution comes with one predefined local token `namePrefix` you must update to a custom value if you intent to leverage the [CI environment](./Getting%20started%20-%20Setup%20environment#31-update-default-nameprefix).
+The solution comes with one predefined local token `namePrefix`. When validating modules through the CI environment, you must update it to a custom value as described in the [Update default nameprefix](./Getting%20started%20-%20Scenario%201%20Onboard%20module%20library%20and%20CI%20environment#31-update-default-nameprefix) paragraph. This is to avoid conflicts with resources requiring a globally unique name, such as storage accounts or key vaults.
```

Comment:
```suggestion
The solution comes with one predefined local token `namePrefix`. When validating modules through the CI environment, you must update it to a custom value as described in the [Update default nameprefix](./Getting%20started%20-%20Scenario%201%20Onboard%20module%20library%20and%20CI%20environment#31-update-default-nameprefix) paragraph. This is done to avoid conflicts with resources requiring a globally unique name, such as storage accounts or key vaults.
```